ipynb/primary-processing/04-prepare_metaanalyses.md

```diff
@@ -40,70 +40,76 @@ We're also going to generate some new files, so let's get them setup.
 >>> ag_pgp_hmp_gg_cleaned_md = agu.get_new_path(agenv.paths['ag-pgp-hmp-gg-cleaned-md'])
 ```
 
-The first step in the process is to merge the the individual tables into larger ones, and then to merge the larger tables into a final one. We're not using QIIME's `parallel_merge_otu_tables.py` here as we also need one of the intermediate tables for subsequent processing.
-
-The first merge we'll do is between the Global Gut and the American Gut.
+We also need to make sure the metadata (the information about the samples) are also merged and consistent. Prior to merge, we're going to add in some additional detail about every sample, such as a column in the mapping file that is the combination of the study title and the body site. We're also going to "generalize" body sites to the type of site they're from (e.g., the back of the hand is just "skin"). This process will also clean the metadata to remove blanks and unknown sample types.
```

Comment:
Very minor, but the usage of the term mapping file comes undefined in this paragraph, and it seems like you were just introducing to the concept of "metadata".